go.mod

```diff
@@ -4,7 +4,7 @@ go 1.18
 
 require (
 	github.com/jackc/pgx/v5 v5.3.1
-	gorm.io/gorm v1.25.0
+	gorm.io/gorm v1.25.2-0.20230530020048-26663ab9bf55
```

Comment:
The latest tag is v1.25.1, 
This tag seems to have been deleted?

Reply:
It points to the master branch since v1.25.1 doesn't have the ErrForeignKeyViolated error.

Reply:
It seems to be a very popular feat, let's create a new tag for GORM? cc @a631807682

Reply:
> It seems to be a very popular feat, let's create a new tag for GORM? cc @a631807682

Adding a tag for gorm alone may cause misunderstanding due to the dependency on the driver implementation. I hope to add a tag to the relevant parts after the driver is implemented.